src/test/java/io/netty/incubator/codec/http3/AbstractHttp3FrameTypeValidationHandlerTest.java

```diff
@@ -126,6 +136,7 @@ protected Http3ErrorCode inboundErrorCodeInvalid() {
 
     @Test
     public void testInvalidTypeOutbound() throws Exception {
+        assumeThat(isOutbound, is(true));
```

Comment:
why not just use `assumeTrue` ?

Reply:
Lemme do that since we are using that elsewhere in this class.